roles/lib/files/FWO.Api.Client/APIcalls/recertification/getOwnedRules.graphql

```diff
@@ -0,0 +1,107 @@
+
+query rulesCertReport(
+  $ownerId: [Int!]
+  $dev: [Int!]
+  $limit: Int
+  $offset: Int
+  $mgmId: [Int!]
+  $relevantImportId: bigint
+  $refdate1: timestamp!
+) {
+  management(
+    where: {
+      mgm_id: { _in: $mgmId }
+      hide_in_gui: { _eq: false }
+      stm_dev_typ: {
+        dev_typ_is_multi_mgmt: { _eq: false }
+        is_pure_routing_device: { _eq: false }
+      }
+    }
+    order_by: { mgm_name: asc }
+  ) {
+    id: mgm_id
+    name: mgm_name
+    devices(
+      where: { hide_in_gui: { _eq: false } }
+      order_by: { dev_name: asc }
+    ) {
+      id: dev_id
+      name: dev_name
+      view_rule_with_owner(
+        limit: $limit
+        offset: $offset
+        where: {
+          _and: [
+            { rule_src: { _nlike: "%Any" } }
+            { rule_dst: { _nlike: "%Any" } }
+            { rule_src: { _neq: "all" } }
+            { rule_dst: { _neq: "all" } }
+            { _or: [{ dev_id: { _eq: 2 } }] }
```

Comment:
don't understand this: for test purposes? Do we need a list of dev-ids as parameter?